frontend/src/pages/Account.vue

```diff
@@ -15,61 +15,212 @@ limitations under the License.
+            <v-list-tile>
+              <v-list-tile-avatar>
+                <v-icon color="teal darken-2">timelapse</v-icon>
+              </v-list-tile-avatar>
+              <v-list-tile-content>
+                <v-list-tile-title class="label">Session</v-list-tile-title>
+                <v-list-tile-sub-title class="content">
+                   Expires {{expiresAt}}
+                </v-list-tile-sub-title>
+              </v-list-tile-content>
+            </v-list-tile>
+          </v-list>
+        </v-card>
+      </v-flex>
+      <v-flex md6 xs12>
+        <v-card>
+          <v-toolbar card dark dense class="teal darken-2">
+            <v-toolbar-title>Access</v-toolbar-title>
+          </v-toolbar>
+          <v-list>
+            <v-list-tile>
+              <v-list-tile-avatar>
+                <v-icon color="teal darken-2">mdi-key </v-icon>
+              </v-list-tile-avatar>
+              <v-list-tile-content>
+                <v-list-tile-title>Token</v-list-tile-title>
+                <v-list-tile-sub-title>Personal bearer token for API authentication</v-list-tile-sub-title>
+              </v-list-tile-content>
+              <v-list-tile-action>
+                <copy-btn :clipboard-text="idToken"></copy-btn>
+              </v-list-tile-action>
+            </v-list-tile>
+            <v-divider inset class="my-2"/>
+            <v-list-tile>
+              <v-list-tile-avatar>
+                <v-icon color="teal darken-2">insert_drive_file</v-icon>
+              </v-list-tile-avatar>
+              <v-list-tile-content>
+                <v-list-tile-title>Kubeconfig</v-list-tile-title>
+              </v-list-tile-content>
+              <v-list-tile-action>
+                <v-tooltip top>
+                  <v-btn slot="activator" icon @click.native.stop="onDownload">
+                    <v-icon>mdi-download</v-icon>
+                  </v-btn>
+                  <span>Download Kubeconfig</span>
+                </v-tooltip>
+              </v-list-tile-action>
+              <v-list-tile-action>
+                <v-tooltip top>
+                  <v-btn slot="activator" icon @click.native.stop="expansionPanel = !expansionPanel">
+                    <v-icon>{{ expansionPanel ? 'expand_less' : 'expand_more' }}</v-icon>
+                  </v-btn>
+                  <span>{{ expansionPanel ? 'Collapse advanced options' : 'Show advanced options' }}</span>
+                </v-tooltip>
+              </v-list-tile-action>
+            </v-list-tile>
+            <v-expand-transition>
+              <v-card v-if="expansionPanel" flat class="mx-2">
+                <v-card-text class="pt-0">
+                  <div class="grey--text text--darken-2">
+                    The downloaded <tt>kubeconfig</tt> will initiate
+                    <a href="https://kubernetes.io/docs/reference/access-authn-authz/authentication/#openid-connect-tokens" target="_blank" class="teal--text text--darken-2">
```

Comment:
Add the arrow icon for links pointing to external pages

Reply:
Added new component `external-link`